app/modules/onboarding/features-usage.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace Elementor\App\Modules\Onboarding;
+
+use Elementor\Tracker;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly
+}
+
+class Features_Usage {
+
+	const ONBOARDING_FEATURES_OPTION = '_elementor_onboarding_features';
+
+	public function register() {
+		if ( ! Tracker::is_allow_track() ) {
+			return;
+		}
+
+		add_filter( 'elementor/tracker/send_tracking_data_params', function ( array $params ) {
+			$params['usages']['onboarding_features'] = $this->get_usage_data();
+
+			return $params;
+		} );
+	}
+
+	public function save_onboarding_features( $data ) {
+		if ( empty( $data ) ) {
+			return;
+		}
+
+		$data = json_decode( $data, true );
```

Comment:
@mykytamurzin Which part of the data specifies this as "hidden fields"?

Reply:
@mykytamurzin Should you check `current_user_can()` or something?

Reply:
@louiswol94 it's already done in add_action(...) here 
https://github.com/elementor/elementor/pull/28193/files#diff-ba434f169932cab25396a4f9ddbdc58723a0eb014ab202d6287882ffccfa5783R486